src/components/common/Extensions/ExtensionDetails/ExtensionDetails.tsx

```diff
@@ -73,6 +74,9 @@ const ExtensionDetails = () => {
     isInstalledExtensionData(extensionData) &&
     extensionData.uninstallable &&
     !extensionData.isDeprecated;
+  // @TODO: Check if the latest network extension version is greater than the current version
+  const canBeUpgraded =
```

Comment:
I'd suggest we make this consistent with the above, so please either rename this to `canExtensionBeUpgraded` or delete "Extension" from the 2 variables above